include/envoy/upstream/upstream.h

```diff
@@ -563,6 +564,7 @@ class PrioritySet {
   COUNTER(upstream_cx_total)                                                                       \
   COUNTER(upstream_cx_tx_bytes_total)                                                              \
   COUNTER(upstream_flow_control_backed_up_total)                                                   \
+  COUNTER(upstream_flow_control_downstream_buffer_overflow_total)                                  \
```

Comment:
Per @antoniovicente I agree that it's really hard to understand what these stats mean from a quick read. Apart from changing these macros to include doc comments, can you add some `/* */` style comments above the flow control stats to help explain what they are?

Reply:
Sure, I'll do that, though haven't seen comments for any of the other stats (other than in the docs).

Reply:
Yes we haven't done these historically because most of these are fairly obvious, but the flow control ones are pretty confusing, and now we have more of them, so we might as well start setting a good example. :)